src/depackers/bunzip2.c

```diff
@@ -132,7 +132,7 @@ static int get_next_block(bunzip_data *bd)
 	unsigned char uc, symToByte[256], mtfSymbol[256], *selectors;
 	unsigned int *dbuf,origPtr;
 
-	hufGroup=NULL;
+	hufGroup=bd->groups; /* silence -Wclobbered */
```

Comment:
This doesn't help. I thought of moving setjmp before the local var assignments.

Reply:
The rest are OK

Reply:
Which CI run was emitting this particular warning? Was it Ubuntu 16.04?

Reply:
My local run - I see it with gcc4.9 and gcc2.95

Reply:
Is the following wrong?
```diff
diff --git a/src/depackers/bunzip2.c b/src/depackers/bunzip2.c
index 8a8a874..83fe918 100644
--- a/src/depackers/bunzip2.c
+++ b/src/depackers/bunzip2.c
@@ -132,13 +132,15 @@ static int get_next_block(bunzip_data *bd)
 	unsigned char uc, symToByte[256], mtfSymbol[256], *selectors;
 	unsigned int *dbuf,origPtr;
 
+	/* Reset longjmp I/O error handling */
+	i=setjmp(bd->jmpbuf);
+	if(i) return i;
+
 	hufGroup=NULL;
 	dbuf=bd->dbuf;
 	dbufSize=bd->dbufSize;
 	selectors=bd->selectors;
-	/* Reset longjmp I/O error handling */
-	i=setjmp(bd->jmpbuf);
-	if(i) return i;
+
 	/* Read in header signature and CRC, then validate signature.
 	   (last block signature means CRC is for whole file, return now) */
 	i = get_bits(bd,24);
```

Reply:
> My local run - I see it with gcc4.9 and gcc2.95

Weird—I just did builds with gcc 4.8.5 (CentOS 7) and 4.1.2 (Debian 4.0 SPARC) and neither came up with that warning. I definitely saw it with the Ubuntu 16.04 workflow, though...

That patch looks OK to me. If it fixes the warning on your end I'll go ahead and switch to your fix.